guacapy/client.py

```diff
@@ -16,7 +16,7 @@
 class Guacamole():
     def __init__(self, hostname, username, password, default_datasource=None,
                  verify=True):
-        self.REST_API = 'https://{}/api'.format(hostname)
+        self.REST_API = 'http://{}/api'.format(hostname)
```

Comment:
Please don't change the connection schema.

Reply:
ok I have modified it back